parquet.tf

```diff
@@ -0,0 +1,223 @@
+          }
+
+          env {
+            name  = "HOST"
+            value = var.host
+          }
+
+          env {
+            name  = "AUTH0_DOMAIN"
+            value = var.auth0_domain
+          }
+
+          env {
+            name  = "AUTH0_CLIENT_ID"
+            value = var.auth0_client_id
+          }
+
+          env {
+            name  = "OIDC_CLIENT_ID"
+            value = var.oidc_client_id
+          }
+
+          env {
+            name = "OIDC_SECRET"
+            value_from {
+              secret_key_ref {
+                name = local.app_name
+                key  = "OIDC_SECRET"
+              }
+            }
+          }
+
+          env {
+            name  = "OIDC_AUTH_METHOD"
+            value = var.oidc_auth_method
+          }
+
+          env {
+            name  = "OIDC_ISSUER"
+            value = var.oidc_issuer
+          }
+
+          env {
+            name  = "REDIS"
+            value = var.redis_connection_string
+          }
+
+          env {
+            name  = "GORILLA_CUSTOM_METRICS_PROVIDER"
+            value = var.cloud_monitoring_connection_string
+          }
+
+          env {
+            name = "WEAVE_SERVICE"
+            value = var.weave_enabled ? "${kubernetes_service.weave.0.metadata.0.name}:9994" : ""
+          }
+
+          env {
+            name = "PARQUET_ENABLED"
+            value = var.weave_enabled ? "true" : "false"
+          }
+
+          env {
+            name = "WEAVE_ENABLED"
+            value = var.weave_enabled ? "true" : "false"
+          }
+
+          dynamic "env" {
+            for_each = var.other_wandb_env
+            content {
+              name  = env.key
+              value = env.value
+
+            }
+          }
+
+          env {
+            name = "GORILLA_STATSD_HOST"
+            value = "datadog.datadog"
+          }
+
+          env {
+            name = "GORILLA_STATSD_PORT"
+            value = "8125"
+          }
+
+          port {
+            name           = "http"
+            container_port = 8087
+            protocol       = "TCP"
+          }
+
+          port {
+            name           = "prometheus"
+            container_port = 8181
+            protocol       = "TCP"
+          }
+
+          resources {
+            requests = {
+              cpu    = "4000m"
+              memory = "8G"
+            }
+            limits   = {
+              cpu    = "8000m"
+              memory = "16G"
+            }
+          }
+        }
+        volume {
+          name = local.app_name
+          config_map {
+            name     = kubernetes_config_map.config_map.metadata[0].name
+            optional = true
+          }
+        }
+      }
+    }
+  }
+
+  timeouts {
+    create = "1h"
+    update = "1h"
+    delete = "10m"
+  }
+}
+
+resource "kubernetes_service" "parquet" {
+  metadata {
+    name = local.parquet_app_name
+  }
+
+  count = var.parquet_enabled ? 1 : 0
+
+  spec {
+    type = "NodePort"
```

Comment:
Cluster IP would probably be better here. I don't think we need to open this port on every node publicly